src/Drupal/DrupalExtension/Manager/DrupalAuthenticationManager.php

```diff
@@ -130,7 +130,10 @@ public function loggedIn()
      */
     public function fastLogout()
     {
-        $this->getSession()->reset();
-        $this->userManager->setCurrentUser(false);
+        $session = $this->getSession();
+        if ($session->isStarted()) {
+            $session->reset();
+            $this->userManager->setCurrentUser(false);
```

Comment:
I wonder if we should unset the user outside the conditional?

Reply:
That makes sense, will update.